src/main/java/firebase/sdk/dashboard/dao/PlatformReleaseDaoDatastore.java

```diff
@@ -0,0 +1,53 @@
+package firebase.sdk.dashboard.dao;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import firebase.sdk.dashboard.data.User;
+import firebase.sdk.dashboard.data.Release;
+import firebase.sdk.dashboard.data.SDKReleaseMetadata;
+import firebase.sdk.dashboard.data.Platform;
+import firebase.sdk.dashboard.data.Platform;
+import java.util.List;
+
+/**
+ * An platform release dao implementation. 
```

Comment:
nit: Include details on what kind of implementation. You could say "Implementation of the Platform release DAO using Datastore for storage."